scripts/build_test.py

```diff
@@ -94,8 +94,7 @@ def test_ensure_files_exist(self):
         non_existent_filepaths = [INVALID_INPUT_FILEPATH]
         # Exception will be raised at first file determined to be non-existent.
         error_message = ('File %s does not exist.') % non_existent_filepaths[0]
-        # When run on Windows, the path contains `\` which should be escaped
-        # in regex.
+        # Escape the special characters in error message.
```

Comment:
> For this one, I think we need to escape the whole message, as there are also special characters like .. But I will update the comment.

Wait, I don't understand. Wouldn't all the special characters in this error message come from the filepath? I don't see anything in "File " or "does not exist." that would be problematic from an escaping perspective.

Reply:
The `.` is also a special character in the regex, but it is trivia, I can change it to just escape the file path.

Reply:
Updated